src/Testcontainers.LocalStack/LocalStackBuilder.cs

```diff
@@ -0,0 +1,102 @@
+namespace Testcontainers.LocalStack;
+
+/// <inheritdoc cref="ContainerBuilder{TBuilderEntity, TContainerEntity, TConfigurationEntity}" />
+[PublicAPI]
+public sealed class LocalStackBuilder : ContainerBuilder<LocalStackBuilder, LocalStackContainer, LocalStackConfiguration>
+{
+    public const ushort LocalStackPort = 4566;
+    public const string LocalStackImage = "localstack/localstack:1.3.1";
+    
+    /// <summary>
+    /// Initializes a new instance of the <see cref="LocalStackBuilder" /> class.
+    /// </summary>
+    public LocalStackBuilder()
+        : this(new LocalStackConfiguration())
+    {
+        DockerResourceConfiguration = Init().DockerResourceConfiguration;
+    }
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="LocalStackBuilder" /> class.
+    /// </summary>
+    /// <param name="dockerResourceConfiguration">The Docker resource configuration.</param>
+    private LocalStackBuilder(LocalStackConfiguration dockerResourceConfiguration)
+        : base(dockerResourceConfiguration)
+    {
+        DockerResourceConfiguration = dockerResourceConfiguration;
+    }
+
+    /// <inheritdoc />
+    protected override LocalStackConfiguration DockerResourceConfiguration { get; }
+    
+    /// <summary>
+    /// Sets the LocalStack external service start port.
+    /// </summary>
+    /// <param name="port">The LocalStack external service start port.</param>
+    /// <returns>A configured instance of <see cref="LocalStackBuilder" />.</returns>
+    public LocalStackBuilder WithExternalServicePortStart(string port)
+    {
+        return Merge(DockerResourceConfiguration, new LocalStackConfiguration(externalServicePortStart: port))
+            .WithEnvironment("EXTERNAL_SERVICE_PORTS_START", port);
```

Comment:
I do not understand when this environment variable is necessary. The configuration does not seem to bind the ports. What exactly connects to the container? Is it container-to-container communication?

Reply:
I unnecessarily followed the previous implementation, it seems that it is not needed because these ports are set by default.

Reply:
Ok, was probably already unnecessary at that time. I think the other language implementations don't use it either.